docs/workshop/ml_pipeline.md

```diff
@@ -14,7 +14,8 @@ Click the Runtimes icon on the left hand side toolbar. Click the ‘+’ on the
 * **Authentication Type:** No Authentication
 * **Cloud Object Storage Endpoint:** S3 Endpoint
 * **Cloud Object Storage Bucket Name:** S3 Bucket Name
-* **Cloud Object Storage Authentication Type:** USER_CREDENTIALS
+* **Cloud Object Storage Authentication Type:** KUBERNETES_SECRET
+* **Cloud Object Storage Credentials Secret:** aiops-tools-workshop-s3
 * **Cloud Object Storage Username:** S3 Access Key ID
```

Comment:
Since we are replacing S3_access_key and s3_secret key with "Aws...access.. " in the .env file. Shall we replace S3 Access Key Id with AWS access key id? wdyt?